public/v4/apps/baikal.yml

```diff
@@ -0,0 +1,26 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: ckulka/baikal:$$cap_version
+        volumes:
+            - '$$cap_appname-data:/var/www/baikal/Specific'
+            - '$$cap_appname-config:/var/www/baikal/config'
+        caproverExtra:
+            containerHttpPort: '80'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: Baikal Version
+          defaultValue: 'nginx'
```

Comment:
This is a mutable tag. Meaning that if you deploy today, you might be installing 0.9.2-nginx, if you deploy tomorrow, you might be installing 0.9.3-nginx. 

We should always use a fixed tag, in this case I'd change this to `0.9.2-nginx`